telemetry/util/heka_message.py

```diff
@@ -15,18 +15,42 @@
 from google.protobuf.message import DecodeError
 
 
-_record_separator = 0x1e
+RECORD_SEPARATOR = 0x1e
 
 
 class BacktrackableFile:
+    """
+    Wrapper for file-like objects that exposes a file-like object interface,
+    but also allows backtracking to the first byte in the file-like object
+    equal to `RECORD_SEPARATOR` that we haven't backtracked to yet.
```

Comment:
The comments is entirely correct as it backtracks to the first byte after `RECORD_SEPARATOR`, right?